launch/launch/launch_service.py

```diff
@@ -316,7 +316,7 @@ def _on_sigint(signum, frame):
 
             def _on_sigterm(signum, frame):
                 # Ignore additional signals until we finish processing this one.
-                prev_handler = signal.signal(signal.SIGINT, signal.SIG_IGN)
+                prev_handler = signal.signal(signal.SIGTERM, signal.SIG_IGN)
```

Comment:
Oooh, now that I recall I think I incidentally fixed that on #210 but it never got in :)